src/features/content-answers/ContentAnswers.tsx

```diff
@@ -10,12 +11,17 @@ const ContentAnswers = () => {
     const [isModalOpen, setIsModalOpen] = useState(false);
 
     const handleClick = () => {
-        setIsModalOpen(!isModalOpen);
+        setIsModalOpen(prevState => !prevState);
```

Comment:
is it possible that it can go from `true` to `false` within this callback ?  The overlay from modal is covering the button while its open.

Reply:
great point! changed to `setIsModalOpen(true)`